setup.py

```diff
@@ -40,11 +40,15 @@
 
 
 # requirements
-from pip.req import parse_requirements
-from pip.download import PipSession
-requirements = list(parse_requirements('requirements.txt', session=PipSession()))
-_requires = [str(r.req.project_name) for r in requirements]
-_install_requires = [str(r.req) for r in requirements]
+try:
+    from pip.req import parse_requirements
+    from pip.download import PipSession
+    requirements = list(parse_requirements('requirements.txt', session=PipSession()))
+    _install_requires = [str(r.req) for r in requirements]
+except:
+    with open('requirements.txt', 'r') as f:
+        for line in f:
+            _install_requires = [line.rstrip('\n') for line in f]
```

Comment:
is there any advantage left to parse `requirements.txt` with pip?

otherwise maybe just removing the pip part reduces the amount of inconsistent behavior?